packages/type-utils/src/TypeOrValueSpecifier.ts

```diff
@@ -219,3 +221,69 @@ export const typeMatchesSomeSpecifier = (
   program: ts.Program,
 ): boolean =>
   specifiers.some(specifier => typeMatchesSpecifier(type, specifier, program));
+
+const getSpecifierNames = (specifierName: string | string[]): string[] => {
+  return typeof specifierName === 'string' ? [specifierName] : specifierName;
+};
+
+const getStaticName = (node: TSESTree.Node): string | undefined => {
```

Comment:
[Non-Actionable] I'm surprised we don't already have a function like this somewhere 😄. I spent a bit searching and couldn't find one. Ah well.